integreat_cms/api/v3/locations.py

```diff
@@ -90,6 +90,8 @@ def transform_poi_translation(poi_translation):
         "opening_hours": opening_hours,
         "location": transform_poi(poi),
         "hash": None,
+        "organization": poi.organization.name if poi.organization else None,
```

Comment:
```suggestion
        "organization": {
            "id": poi.organization.id,
            "slug": poi.organization.slug,
            "name": poi.organization.name,
            "logo": poi.organization.icon.url if poi.organization.icon else None,
        }
        if poi.organization
        else None,
```
I can imagine that it might become helpful in the future to deliver all these additional fields as well in case the apps want to display more than just the organization's name or want to enable linking between all pois which belong to the same organization.